subsystems/csr/csrsrv/srv.h

```diff
@@ -48,9 +48,9 @@
 #define CSR_PORT_NAME       L"ApiPort"
 #define UNICODE_PATH_SEP    L"\\"
 
-#define ROUND_UP(n, align) ROUND_DOWN(((ULONG)n) + (align) - 1, (align))
-#define ROUND_DOWN(n, align) (((ULONG)n) & ~((align) - 1l))
-#define InterlockedIncrementUL(Value) _InterlockedIncrement((PLONG)Value)
-#define InterlockedDecrementUL(Value) _InterlockedDecrement((PLONG)Value)
+#define ROUND_UP(n, align)              ROUND_DOWN(((ULONG)(n)) + (align) - 1, (align))
+#define ROUND_DOWN(n, align)            (((ULONG)(n)) & ~((align) - 1l))
```

Comment:
I don't see why adding those braces around the n does add to safety or readability in any way. In other words: touching the ROUND_UP and ROUND_DOWN lines is superfluous in this PR.